Tweaks to the READMEs

johnsca opened the pull request:
Updates README for charms and bundle.

camille-rodriguez commented on bundle/README.md:
````diff
@@ -1,3 +1,26 @@
+via a [bundle overlay][] or a subsequent command:
+
+```sh
+juju config metallb-controller iprange=<IPRANGE>
````
Could you adapt this with how it was updated for the docs? i.e 
"192.168.1.240-192.168.1.250" instead of IPRANGE

And add the following line:
Multiple IP pools can be specified as well, if using a CIDR notation delimited by a comma:

juju config metallb-controller iprange="192.168.1.240/28, 10.0.0.0/28"

Thank you! I should have done it myself in the main readme.

evilnick replied:
> And add the following line:
> Multiple IP pools can be specified as well, if using a CIDR notation delimited by a comma:
> 
> juju config metallb-controller iprange="192.168.1.240/28, 10.0.0.0/28"

Yes, if we are going to include an example, it should be a useful one.

camille-rodriguez reviewed (changes requested):
One little tweak for the IP range please, otherwise it's great!

evilnick commented:
I understand the desire to have some content here, but I think the Link to the upstream docs should be more prominent. Until there is a plan for maintaining this info both in the docs and in the README, there is going to be potential for things to be unmaintained in one place or the other. Updating the README for charms in the past has proved problematic.

I will make a slight tweak to the words

evilnick commented:
The other thing is there is a chicken/egg situation with the docs, which can't be published until 1.19

kwmonroe commented on bundle/README.md:
```diff
@@ -1,3 +1,39 @@
+The official documentation for these charms and how to use them with Kubernetes
+can be found at <https://ubuntu.com/kubernetes/docs/metallb>.
+
+## Deploying
```
This `Deploying` section is incomplete/confusing.  You can't deploy ~containers/metallb outside of a k8s cloud, which means we'd have to duplicate https://github.com/charmed-kubernetes/kubernetes-docs/pull/472 here to ensure people know what steps need to happen before this charm is expected to work.

IMHO, the `Overview` is enough.  Leave these deployment / config steps out in favor of the official docs.

kwmonroe reviewed (approved):
lgtm

johnsca commented:
It turns out that Juju is smart enough to still use resources from the charm store when you use an overlay to override the charm to point to a local copy:

```
$ juju deploy ./bundle --overlay ./docs/local-overlay.yaml
Executing changes:
- upload charm /home/johnsca/juju/charms/metallb-operator/build/metallb-controller.charm for series kubernetes
- deploy application metallb-controller with 1 unit on kubernetes using /home/johnsca/juju/charms/metallb-operator/build/metallb-controller.charm
  added resource metallb-controller-image
- upload charm /home/johnsca/juju/charms/metallb-operator/build/metallb-speaker.charm for series kubernetes
- deploy application metallb-speaker with 1 unit on kubernetes using /home/johnsca/juju/charms/metallb-operator/build/metallb-speaker.charm
  added resource metallb-speaker-image
Deploy of bundle completed.
```

evilnick reviewed (approved):
:+1:
